miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -336,18 +335,10 @@ public String poolConcentrationUnits() {
   };
 
   private void populateSampleClasses(ModelMap model) throws IOException {
-    List<SampleClass> sampleClasses = new ArrayList<>();
-    List<SampleClass> tissueClasses = new ArrayList<>();
-    Collection<SampleValidRelationship> relationships = sampleValidRelationshipService.getAll();
-    // Can only create Tissues, Stocks, and Aliquots from this page, so remove other classes
-    for (SampleClass sc : sampleClassService.getAll()) {
-      if (SampleTissue.CATEGORY_NAME.equals(sc.getSampleCategory())) {
-        tissueClasses.add(sc);
-      }
-      if (sc.hasPathToIdentity(relationships)) {
-        sampleClasses.add(sc);
-      }
-    }
+    List<SampleClass> sampleClasses = new ArrayList<>(sampleClassService.getAll());
+    List<SampleClass> tissueClasses = sampleClasses.stream()
+        .filter(sc -> SampleTissue.CATEGORY_NAME.equals(sc.getSampleCategory()))
+        .collect(Collectors.toList());
```

Comment:
Attempting to receive LCM Tubes results in the error described below (line 808): `Row 1: Tissue Processing class with id 10 has no Tissue parents`. It shouldn't be possible to enter data for sample classes for which the hierarchy can't correctly be inferred.